doc/source/reference/apis/metadata.md

```diff
@@ -15,18 +15,23 @@ Incubating update 1:
 - we added `v1` format that better describes current SeldonMessage
 - definition through environmental variables now accepts both `yaml` and `json` input
 
+Incubating update 2:
+- we added GRPC support for both Model and Graph metadata
+- adjustments to `v1` (it is now also an array of inputs/outputs metadata) and removed explicit distinction
 
+We plan to graduate metadata features with the 1.3 release of Seldon Core.
 
 ## Examples
-- [Basic Examples for Model with Metadata](../../examples/metadata.html)
-- [SKLearn Server example with MinIO](../../examples/minio-sklearn.html)
-- [Deployment Level Metadata](../../examples/graph-metadata.html).
+- [Model Metadata (examples)](../../examples/metadata.html)
+- [Model Metadata (format examples)](../../examples/metadata_schema.html)
```

Comment:
I don't think we shoudl be distinguishing model metadata from deployment metadata, we can have all in a single example with different HTTP calls

Reply:
Maybe just a notebook with metadata rest and one with metadata GRPC, showing how you can query one of the models vs the graph with different request paths